docs/developer-docs/latest/setup-deployment-guides/configurations/required/databases.md

```diff
@@ -19,31 +19,21 @@ The CLI installation guide details [supported database and versions](http://loca
-      - `client` (string): Database client to create the connection. `sqlite` or `postgres` or `mysql`.
-      - `host` (string): Database host name. Default value: `localhost`.
-      - `port` (integer): Database port.
-      - `database` (string): Database name.
-      - `username` (string): Username used to establish the connection.
-      - `password` (string): Password used to establish the connection.
-      - `timezone` (string): Set the default behavior for local time. Default value: `utc` [Timezone options](https://www.php.net/manual/en/timezones.php).
-      - `schema` (string): Set the default database schema. **Used only for Postgres DB.**
-      - `ssl` (boolean/object): For ssl database connection. Object is used to pass certificate files as strings.
-    - `options` Options used for database connection.
-      - `debug` (boolean): Show database exchanges and errors.
-      - `autoMigration` (boolean): To disable auto tables/columns creation for SQL database.
-      - `pool` Options used for database connection pooling. For default value and more information, look at [Knex's pool config documentation](https://knexjs.org/#Installation-pooling).
-        - `min` (integer): Minimum number of connections to keep in the pool.
-        - `max` (integer): Maximum number of connections to keep in the pool.
-        - `acquireTimeoutMillis` (integer): Maximum time in milliseconds to wait for acquiring a connection from the pool.
-        - `createTimeoutMillis` (integer): Maximum time in milliseconds to wait for creating a connection to be added to the pool.
-        - `idleTimeoutMillis` (integer): Number of milliseconds to wait before destroying idle connections.
-        - `reapIntervalMillis` (integer): How often to check for idle connections in milliseconds.
-        - `createRetryIntervalMillis` (integer): How long to idle after a failed create before trying again in milliseconds.
+- `connection` (object): Database configuration options
+  - `client` (string): Database client to create the connection. `sqlite` or `postgres` or `mysql`.
+  - `connection` (object): Database connection information
+    - `host` (string): Database host name. Default value: `localhost`.
+    - `port` (integer): Database port.
+    - `database` (string): Database name.
+    - `username` (string): Username used to establish the connection.
+    - `password` (string): Password used to establish the connection.
+    - `timezone` (string): Set the default behavior for local time. Default value: `utc` [Timezone options](https://www.php.net/manual/en/timezones.php).
+    - `schema` (string): Set the default database schema. **Used only for Postgres DB.**
+    - `ssl` (boolean/object): For ssl database connection. Object is used to pass certificate files as strings.
+  - `debug` (boolean): Show database exchanges and errors.
+  - `autoMigration` (boolean): To disable auto tables/columns creation for SQL database.
```

Comment:
this option doesn't exist anymore but the pool options are still available

Reply:
How do we disable the migration process?
And at what level do the pool settings apply?

Reply:
all that is in the first connection key is passed down to knex. you can just follow their doc on that. 
Right now it's not possible to disable the migration process. We are thinking about it